### PR TITLE
Fixing renovate template issues

### PR DESCRIPTION
**Pull request by tomahawk28**
1. Down the version limit of k8s to 0.21.0 
2. `--reviewer` is not required for `devctl gen renovate` anymore.  

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/gen/input/renovate/internal/file/renovate.json.template
Comment (tomahawk28) [inline pkg/gen/input/renovate/internal/file/renovate.json.template:8]: I think using extends for reviewers is more cleaner.

Comment (rossf7) [reply pkg/gen/input/renovate/internal/file/renovate.json.template:8]: How is reviewer set if there is no flag?

**Review by rossf7 — APPROVED** [review]
Fine with me. Just a query on how extends works.

File: pkg/gen/input/renovate/internal/file/renovate.json.template
Comment (tomahawk28) [reply pkg/gen/input/renovate/internal/file/renovate.json.template:8]: It would assign CODEOWNER as a reviewer automatically. I think it makes sense to loose the restriction on the reviewer flag when it has been not working properly from the renovate side.

**Comment by tomahawk28** [comment]
Just a small fix :) I'm going without approval from team-biscuit.

**Pull request merged**
